capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
```

Comment:
Hi there,

Firstly, I want to congratulate you on the progress you've made so far! 👏 Your implementation of the `CapitalizeDecorator` class is a clear indication of your understanding of Ruby's inheritance and the Decorator pattern. Great job on getting the inheritance from the `Decorator` class set up correctly! 🎉

However, I did notice a small discrepancy with the previous review guidelines concerning the `correct_name` method. To align with the requested changes, we should directly interact with the `@nameable` instance variable within your `CapitalizeDecorator` class.

Here's how you can refine your `correct_name` method:

```ruby
class CapitalizeDecorator < Decorator
  def correct_name
    @nameable.correct_name.capitalize
  end
end
```

This change ensures we are directly calling the `correct_name` method on the `@nameable` instance, which is what was required by the previous reviewer and also the project requirements..

Keep up the good work, and let me know if you have any questions regarding this change!